020-Connect/020-JavaScript/020-prisma.mdx

```diff
@@ -75,7 +75,12 @@ const result = await prisma.users.findMany({
 });
 ```
 
+## Shadow Database
+
+Certain Prisma commands such as `prisma migrate dev` require an additional [shadow database](https://www.prisma.io/docs/orm/prisma-migrate/understanding-prisma-migrate/shadow-database).
```

Comment:
```suggestion
Migrations with Prisma require a [shadow database](https://www.prisma.io/docs/orm/prisma-migrate/understanding-prisma-migrate/shadow-database).
```

Reply:
@SferaDev It's only `dev` migrations that require the shadow db according to their docs, so I didn't want to imply it is needed for all commands:

```
The shadow database is not required in production, and is not used by production-focused commands such as prisma migrate resolve and prisma migrate deploy.
```

Let me know if you still think we need this rephrasing, happy to merge if so.

Reply:
It's not needed for all Prisma CLI commands, but it is necessary for migration building. It's better to default to saying that they need the shadow database instead of making them think they don't need it and once a command fails they need to create it.